src/core/utils/helpers.ts

```diff
@@ -135,3 +135,16 @@ export const dateMatchesUsFormat = (date: string | null) => {
   const returnValue = dateFound && dateFound.length > 0 ? dateFound[0] : null;
   return returnValue;
 };
+
+export const usDateStringToEurDateObj = (date: string | null | undefined) => {
+  if (!date || date === null) {
+    return date;
+  }
```

Comment:
Consider if you can drop the ` | null | undefined` and the following check. It seems as if you can move the call to `usDateStringToEurDateObj` to `ReviewMetadata` and let the `date` argument` accept `string | null | undefined`.

Also: I am not sure there is any such thing as a Eur(o) date object. It's just a date object.

Reply:
👍